apa-annotated-bibliography.csl

```diff
@@ -413,7 +496,15 @@
       <else-if type="legal_case">
         <group prefix=" (" suffix=")" delimiter=" ">
           <text variable="authority"/>
-          <date variable="issued" form="text"/>
+	  <choose>
+	    <if variable="container-title" match="any">
+	      <!--Only print year for cases published in reporters-->
+	      <date variable="issued" form="numeric" date-parts="year"/>
+	    </if>
+	    <else>
+              <date variable="issued" form="text"/>
+	    </else>
+	  </choose>
```

Comment:
@adam3smith, there are a bunch of redundant `match="any"`  attributes on elements with a single test.